gdal/ogr/ogrsf_frmts/shape/shpopen.c

```diff
@@ -266,6 +271,8 @@
  */
 
 #include "shapefil.h"
+#include "cpl_conv.h"
```

Comment:
This file is originally from the shapelib project which doesn't depend on GDAL CPL library. It would be great if that could remain independant, so the config option should be probably queried in the OGR specific files and set through a function in this file.